src/style/uomthesis.sty

```diff
@@ -213,7 +213,7 @@
 				{\@university} \par
 					\vspace{2cm}
 
-					Submitted in total fulfilment of the requirements of the degree\par
+					Submitted in Total Fulfillment of the Requirements of the Degree of Doctor of Philosophy \par

```

Comment:
(Looks like the new sample PDF is missing this change.)

The requirements don't specify title case here; it looks a bit loud in final print.  I'd recommend sticking to sentence-case.